README.md

```diff
@@ -14,9 +14,11 @@ This repository houses the definitions for jobs on our Jenkins instance. They ar
 | docker_build-safe_cli_libs_build_container    | Constructs a new build container for safe-cli. |
 | docker_build-safe_nd_libs_build_container     | Constructs a new build container for safe-nd.  |
 | pipeline-safe_cli                             | Release pipeline for [safe-cli](https://github.com/maidsafe/safe-nd). This pipeline will build and test on Linux, Windows and macOS and deploy artifacts to GitHub releases. |
+| pipeline-safe-nd                              | Release pipeline for [safe-nd](https://github.com/maidsafe/safe-nd). Currently this release process only has a simple build and test stage. |
```

Comment:
looks like this has been added in twice now

Reply:
Oops, sorry, not sure how that happened. Pushed a fix.

Cheers.